Demo/Demo/ViewModels/VenmoPaymentsViewModel.swift

```diff
@@ -1,3 +1,31 @@
+import CorePayments
 import Foundation
 
-class VenmoPaymentsViewModel: ObservableObject { }
+class VenmoPaymentsViewModel: ObservableObject {
+    
+    let configManager = CoreConfigManager(domain: "Venmo Payments")
+    
+    @Published var state = VenmoState()
+    
+    func getEligibility() async throws {
+        DispatchQueue.main.async {
+            self.state.isVenmoEligibleResponse = .loading
+        }
+        do {
+            let config = try await configManager.getCoreConfig()
+            let eligibilityRequest = EligibilityRequest(currencyCode: "USD", intent: .CAPTURE)
```

Comment:
Take it or leave it - maybe we want to leave a TODO for this, but this intent will have to be the same intent used to create the orderID which will eventually get used when constructing a `VenmoCheckoutRequest(orderID: "<ORDER_ID>")`

Reply:
i've updated it by adding the intent as a parameter and making changes to the UI: 8f253cf](https://github.com/paypal/paypal-ios/pull/286/commits/8f253cf4ef60fd97edb68398f19a0c96f576eaae)